src/Stryker.Core/Stryker.Core/Reporters/DashboardReporter.cs

```diff
@@ -39,13 +39,17 @@ public void OnAllMutantsTested(IReadOnlyProjectComponent reportComponent)
 
             if (reportUri != null)
             {
-                _logger.LogDebug("Your stryker report has been uploaded to: \n {0} \nYou can open it in your browser of choice.", reportUri);
-                _consoleWriter.Write(Output.Green($"Your stryker report has been uploaded to: \n {reportUri} \nYou can open it in your browser of choice."));
-
                 if (_options.ReportTypeToOpen == Options.Inputs.ReportType.Dashboard)
                 {
                     _processWrapper.Open(reportUri);
                 }
+                else
+                {
+                    _consoleWriter.Write(Output.Cyan("Hint: by passing \"--open dashboard or -o dashboard\" the report will open automatically once stryker is done."));
```

Comment:
Is this correct? Per [this](https://github.com/stryker-mutator/stryker-net/pull/1795/files#diff-17ed18489a956f326ec0fe4040850c5bc9261d4631fb42da4c52891d74a59180) it should be something like `--open-report:dashboard`.

Reply:
```suggestion
                    _consoleWriter.Write(Output.Cyan("Hint: by passing \"--open dashboard or -o dashboard\" the report will open automatically once Stryker is done."));
```

Reply:
My bad 🙈 apparently I don't know Stryker as well as you do haha